src/main/resources/changesets/changelog-consent-97.0.xml

```diff
@@ -0,0 +1,56 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet id="97.0" author="smarks">
+      <renameColumn tableName="election"
+                    oldColumnName="electionid"
+                    newColumnName="election_id"/>
+
+      <renameColumn tableName="election"
+                    oldColumnName="createdate"
+                    newColumnName="create_date"/>
+
+      <renameColumn tableName="election"
+                    oldColumnName="referenceid"
+                    newColumnName="reference_id"/>
+
+      <renameColumn tableName="election"
+                    oldColumnName="lastupdate"
+                    newColumnName="last_update"/>
+
+      <renameColumn tableName="election"
+                    oldColumnName="finalaccessvote"
+                    newColumnName="final_access_vote"/>
+
+      <renameColumn tableName="election"
+                    oldColumnName="userestriction"
+                    newColumnName="use_restriction"/>
+
+      <renameColumn tableName="election"
+                    oldColumnName="translateduserestriction"
+                    newColumnName="translated_use_restriction"/>
```

Comment:
FYI, these columns are unused. I believe that at one point, we stored the value of the consent use restriction at the time of election creation so that if the consent changed later, the election would still be for the previous version. I can't find any usages of these in the codebase so will file a separate ticket to remove these columns